python/llm/example/GPU/PyTorch-Models/Model/llama2/README.md

```diff
@@ -67,3 +67,22 @@ What is AI?
 
 AI, or artificial intelligence, refers to the ability of machines to perform tasks that would typically require human intelligence, such as learning, problem-solving,
 ```
+
+## Example 2 - Low Memory Version: Predict Tokens using `generate()` API
+
+Sometimes your GPU does not have enough memory to load the entire 4-bit model in your GPU (e.g. [meta-llama/Llama-2-70b-chat-hf](https://huggingface.co/meta-llama/Llama-2-70b-chat-hf)). In the example [low_memory_generate.py](./low_memory_generate.py), we show one way of how to load Llama models with very low GPU memory consumption. However this could be much slower than the original way. The implementation is adapted from [here](https://www.kaggle.com/code/simjeg/platypus2-70b-without-wikipedia-rag).
```

Comment:
If you're not able to load the full 4-bit model (e.g. [meta-llama/Llama-2-70b-chat-hf](https://huggingface.co/meta-llama/Llama-2-70b-chat-hf)) in one GPU as shown in [Example 1](add the link to Example 1 here), you may try this example instead. In [low_memory_generate.py](./low_memory_generate.py), we show a way to load very large models with very low GPU memory footprint. However this could be much slower than the standard way. The implementation is adapted from [here](https://www.kaggle.com/code/simjeg/platypus2-70b-without-wikipedia-rag).

Reply:
Done.